packages/xen-api/src/index.js

```diff
@@ -153,6 +156,10 @@ export class Xapi extends EventEmitter {
     }
   }
 
+  get httpProxyAgent() {
```

Comment:
It can be used to handle other things (like `allowUnauthorized`, maybe in a future change).

```suggestion
  get httpAgent() {
```